js/map.js

```diff
@@ -0,0 +1,88 @@
+/* global L:readonly */
+
+import {disableElements, enableElements} from './disable.js';
+import {similarListings} from './listing.js';
+import {
+  FLOAT_DIGITS,
+  TOKYO_COORDINATES,
+  ZOOM,
+  MAIN_PIN,
+  PIN
+} from './data.js';
+import {createSimilarPopup} from './new-cards.js';
+
+const addressField = document.querySelector('#address');
+
+if (window.L) {
+  const map = L.map('map-canvas', {
+    dragging: !L.Browser.mobile,
+    tap: false,
+  });
+
+  const mainPinIcon = L.icon({
+    iconUrl: '../img/main-pin.svg',
```

Comment:
```suggestion
    iconUrl: 'img/main-pin.svg',
```
тоже путь относительно index.html, а не js-ки